Weather/ContentView.swift

```diff
@@ -8,9 +8,18 @@
 import SwiftUI
 
 struct ContentView: View {
+    @ObservedObject var cityVM = CityViewModel()
+    
     var body: some View {
-        Text("Hello, world!")
-            .padding()
+        ZStack(alignment: .bottom) {
+            VStack(spacing: 0) {
+                MenuHeaderView(cityViewModel: cityVM)
+                ScrollView(showsIndicators: false) {
+                    CityView(cityVM: cityVM)
+                }
+            }.padding(.top, 50)
```

Comment:
I would suggest you to call methods on a new line. It's more readable in my opinion:
```
}
.padding(.top, 50)
```

Reply:
Fixed.)